src/main/java/org/opentripplanner/model/Stop.java

```diff
@@ -117,17 +117,20 @@ public static Stop stopForTest(
    */
   public static Stop stopForTest(
     String idAndName,
-    String desc,
+    I18NString desc,
     double lat,
     double lon,
     Station parent,
     WheelchairBoarding wheelChairBoarding
   ) {
+    NonLocalizedString nonLocalizedDesc = desc != null
+      ? new NonLocalizedString(desc.toString())
```

Comment:
You can use `NonLocalizedString#ofNullable` for these

Reply:
This will be done.

Reply:
Fixed.